vnext/Microsoft.ReactNative.Managed.UnitTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Microsoft.ReactNative.Managed.UnitTests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Microsoft.ReactNative.Managed.UnitTests")]
+[assembly: AssemblyCopyright("Copyright ©  2019")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
```

Comment:
>emblyCulture("")] [](start = 14, length = 17)

Can we just remove all empty fields?

Reply:
This is an auto-generated file. I did not pay much attention to it. I will revisit and clean it up at some point.

---
In reply to: [356912075](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356912075) [](ancestors = 356912075)